bin/pgenv

```diff
@@ -1416,17 +1438,29 @@ EOF
         fi
 
         title="PostgreSQL version $v"
+        configuration_file=$( pgenv_configuration_file_name $v )
 
         case $action in
             show)
                 pgenv_configuration_dump_or_exit "$v" "$title" ;;
-            write)
+            init)
+                # check that the file does not exist, so
+                # that the program does not re-initialize it
+                if [ -f "$configuration_file" ]; then
+                    pgenv_debug "Cannot init [$configuration_file] for version [$v]: file already existing"
+                    echo "Configuration file [$configuration_file] already exists, cannot re-init it!"
```

Comment:
This seems redundant.

Reply:
the first line is about a debug message, and displays the effective version (either provided by the user or computed by the function), the second is a normal message and does not include the `$v` version. I've reduced in f25e7ca.

Reply:
Ooh, gotcha.